.github/workflows/build.yml

```diff
@@ -225,7 +225,6 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: "14"
-          cache: "yarn"
```

Comment:
I'm getting the error 'Error: getCacheEntry failed: Cache service responded with 503' for 3 out of 4 runs. Seems like it's an ongoing bug - see https://github.com/bahmutov/npm-install/issues/80. To date the only solution is to remove cacheing 😞